src/System.Private.ServiceModel/tools/setupfiles/BuildWCFTestService.cmd

```diff
@@ -39,6 +39,7 @@ call :build %*
 goto :AfterBuild
 
 :build
+msbuild %~dp0..\test\Bridge.Build.Tasks\Bridge.Build.Tasks.sln
```

Comment:
Is this supposed to be in there? It's missing parameters such as /maxcpucount and logging to a build log.